packages/collector/test/tracing/messaging/node-rdkafka/test.js

```diff
@@ -52,13 +52,11 @@ const SINGLE_TEST_PROPS = {
 const retryTime = 1000;
 const topic = 'rdkafka-topic';
 
-let mochaSuiteFn;
-
-if (!supportedVersion(process.versions.node) || semver.lte(process.versions.node, '16.0.0')) {
-  mochaSuiteFn = describe.skip;
-} else {
-  mochaSuiteFn = describe;
-}
+// TODO: investigate as part of https://jsw.ibm.com/browse/INSTA-17132
+const mochaSuiteFn =
```

Comment:
IMO, we should wait for v23 to be released, and if this case fails, we can go ahead and merge this PR directly.

Reply:
rdkafka has 45 days support policy. 

IMO no need to fix this as part of the Node v23 support.

But we can investigate straight after the release.